### PR TITLE
Update Bottom Sheet

### PR DESCRIPTION
**Pull request by ThusSpokeNomad**
# 📝 Why & how

Update bottom sheet (as requested by @brentvatne over here: https://github.com/reactwg/react-native-new-architecture/discussions/167#discussioncomment-10056394)

# ✅ Checklist

- [ ] Added library to **`react-native-libraries.json`**
- [x] Updated library in **`react-native-libraries.json`**

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [ ] Documented in this PR how you fixed or created the feature.

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -11224,15 +11224,24 @@
...
     "ios": true
+  },
+  {
+    "githubUrl": "https://github.com/gorhom/react-native-bottom-sheet",
+    "examples": [ "https://github.com/gorhom/react-native-bottom-sheet/tree/master/example" ],
+    "images": [ "https://raw.githubusercontent.com/gorhom/react-native-bottom-sheet/master/preview.gif" ],
+    "npmPkg": "@gorhom/bottom-sheet",
+    "android": true,
+    "ios": true,
+    "newArchitecture": false
```
Comment (ThusSpokeNomad) [inline react-native-libraries.json:11245]: ^ this is the new entry.

**Comment by Simek** [comment]
Hey @ThusSpokeNomad, looks like something lead to the reformatting in the whole data file. Usually, we would need to revert those changes, but in this case, it looks like we can close the PR, since given library is already present in the directory:
* https://reactnative.directory/?search=%40gorhom%2Fbottom-sheet

We have a deduplicate mechanism, which should fail the CI, but it did not work in this case for some reason, going to investigate the checks.

**Comment by ThusSpokeNomad** [comment]
hey @Simek nice catch, I didn't see that!
let's consider this as an update then?
I also reverted formatting changes too.

**Review by Simek — APPROVED** [review]
Sounds good, thanks for the updates! 👍

**Pull request merged**